Makefile

```diff
@@ -1,227 +1,120 @@
-clean_triggers:
-	rm -rf $(trigger_build) triggers/.daml
-
-$(operator_pid): |$(state_dir) $(trigger_build)
-	(daml trigger --dar $(trigger_build) \
-	    --trigger-name OperatorTrigger:handleOperator \
-	    --ledger-host localhost --ledger-port 6865 \
-	    --ledger-party Operator > $(operator_log) & echo "$$!" > $(operator_pid))
-
-start_operator: $(operator_pid)
-
-stop_operator:
-	pkill -F $(operator_pid); rm -f $(operator_pid) $(operator_log)
-
-$(ccp_pid): |$(state_dir) $(trigger_build)
-	(daml trigger --dar $(trigger_build) \
-	    --trigger-name ClearingTrigger:handleClearing \
-	    --ledger-host localhost --ledger-port 6865 \
-	    --ledger-party Ccp > $(ccp_log) & echo "$$!" > $(ccp_pid))
-
-start_ccp: $(ccp_pid)
-
-stop_ccp:
-	pkill -F $(ccp_pid); rm -f $(ccp_pid) $(ccp_log)
-
-### DA Marketplace Auto-Approve Triggers (all parties)
-
-start_autoapprove: |$(state_dir) $(trigger_build)
-	./scripts/run-triggers.sh $(trigger_build) $(state_dir)
-
-stop_autoapprove: |$(state_dir)
-	./scripts/stop-triggers.sh $(state_dir)
-
-### DA Marketplace Custodian Bot
-
-$(custodian_pid): |$(state_dir) $(trigger_build)
-	(daml trigger --dar $(trigger_build) \
-	    --trigger-name CustodianTrigger:handleCustodian \
-	    --ledger-host localhost --ledger-port 6865 \
-	    --ledger-party Custodian > $(custodian_log) & echo "$$!" > $(custodian_pid))
+$(trigger_dar): $(trigger_dar_src) $(PKG_DIR)
+	cp $(trigger_dar_src) $@
 
-start_custodian: $(custodian_pid)
+# Codegen target
+$(damljs): $(dar_src)
+	daml codegen js $(dar_src) -o $@
 
-stop_custodian:
-	pkill -F $(custodian_pid); rm -f $(custodian_pid) $(custodian_log)
-
-### DA Marketplace Broker Bot
-
-$(broker_pid): |$(state_dir) $(trigger_build)
-	(daml trigger --dar $(trigger_build) \
-	    --trigger-name BrokerTrigger:handleBroker \
-	    --ledger-host localhost --ledger-port 6865 \
-	    --ledger-party Broker > $(broker_log) & echo "$$!" > $(broker_pid))
-
-start_broker: $(broker_pid)
-
-stop_broker:
-	pkill -F $(broker_pid); rm -f $(broker_pid) $(broker_log)
-
-
-### DA Marketplace Exchange Bot
-
-$(exchange_pid): |$(state_dir) $(trigger_build)
-	(daml trigger --dar $(trigger_build) \
-	    --trigger-name AutoApproval:autoApprovalTrigger \
-	    --ledger-host localhost --ledger-port 6865 \
-	    --ledger-party Exchange > $(exchange_log) & echo "$$!" > $(exchange_pid))
-
-start_exchange: $(exchange_pid)
-
-stop_exchange:
-	pkill -F $(exchange_pid); rm -f $(exchange_pid) $(exchange_log)
-
-### DA Marketplace <> Exberry Adapter
-$(exberry_adapter_dir):
+# Exberry Adapter target
+$(exberry_adapter): $(PKG_DIR)
 	cd exberry_adapter && $(PYTHON) setup.py sdist
 	rm -fr exberry_adapter/marketplace_exchange_adapter.egg-info
+	cp exberry_adapter/dist/$(exberry_adapter_name) $@
 
-$(adapter_pid): |$(state_dir) $(exberry_adapter_dir)
-	cd exberry_adapter && pipenv install && (DAML_LEDGER_URL=localhost:6865 $(PYTHON) bot/exberry_adapter_bot.py > ../$(adapter_log) & echo "$$!" > ../$(adapter_pid))
```

Comment:
I think we should retain this, or add it to the `scripts` folder - maybe also add a section in the local development README about running the adapter/integration? we could always do this in a later PR

Reply:
Good catch, added that back